<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Einar Uvsl√∏kk, &lt;einar.uvslokk@linux.com&gt;</author>
 <class>LoggerWidget</class>
 <widget class="QWidget" name="LoggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTextEdit" name="messageEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Display message types:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="errorBox">
       <property name="text">
        <string>Errors</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="debugBox">
       <property name="text">
        <string>Debug</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="infoBox">
       <property name="text">
        <string>Info</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>141</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string>Clear log</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>LoggerWidget</receiver>
   <slot>clearLogger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>debugBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LoggerWidget</receiver>
   <slot>rebuildLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>errorBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LoggerWidget</receiver>
   <slot>rebuildLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LoggerWidget</receiver>
   <slot>rebuildLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rebuildLog()</slot>
 </slots>
</ui>
